ironfish-rust-nodejs/src/structs/transaction.rs

```diff
@@ -33,6 +36,18 @@ pub struct NativeTransactionPosted {
 #[napi]
 pub const PROOF_LENGTH: u32 = PROOF_SIZE;
 
+#[napi]
+pub const TRANSACTION_SIGNATURE_LENGTH: u32 = TRANSACTION_SIGNATURE_SIZE;
```

Comment:
Is this being used? I see `TRANSACTION_SIGNATURE_SIZE` but I don't think it's being referenced outside of exports

Reply:
I want to define in the transaction mod.rs so the constant is in the same file as the original definition. If I define in ts code, there is a gap. When people change the transaction class, it might not be aware of this constant in ts.